### DISCUSSION TITLE
Organization rename to hygieia (lower case)

### DISCUSSION THREAD

**Discussion by rvema** [hygieia/hygieia#3341] [Ideas]
w.r.t. #3218  as suggested by @friederbluemle , I feel its a good idea to rename the Github organization to lowercase i.e. `hygieia` instead of `Hygieia`. @tmbjmu @tabladrum @nireeshT - Please pour in your thoughts

**Comment by friederbluemle** [comment]
Thanks for starting the discussion @rvema :+1:

**Comment by friederbluemle** [comment]
Actually, the details about this discussion are part of #3218 (not #3217). Can you update the first post please @rvema?

**Comment by rvema** [comment]
@friederbluemle The Organization has been renamed to `hygieia` . We will work on renaming some of the underlying repos to comply with the lower case standard.
